cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/sla/DynamicVmCreationByCpuUtilizationAndFreePesOfVm.java

```diff
@@ -331,17 +337,18 @@ private Host createHost() {
     }
 
     /**
-     * Creates a Vm object.
+     * Creates a VM object.
      *
-     * @return the created Vm
+     * @return the created VM
      */
     private Vm createVm() {
         final int id = createsVms++;
         final int pes = VM_PES[(int) randVm.sample()];
 
-        Vm vm = new VmSimple(id, 1000, pes)
+        Vm vm = new VmSimple(id, vmMips, pes)
```

Comment:
Here you are creating all VMs with the same MIPS, that is OK. But if you change this code here, the code I commented above will be wrong.
Here you have just to define the vmMips attribute as a constant (static final), and rename to VM_MIPS.